src/main/java/seedu/address/logic/parser/alias/UnaliasCommandParser.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser.alias;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ALIAS;
+
+import seedu.address.logic.commands.alias.UnaliasCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class UnaliasCommandParser implements Parser<UnaliasCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the UnaliasCommand
+     * and returns an UnaliasCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
Should there be a blank line between description and `@throws`?

Saw this in other files.